README.md

```diff
@@ -338,32 +338,46 @@ GET /api/v1/users/:user_id/job_applications/:job_application_id
 
 Successful Response:
 ```
-Status: 200 OK
-{:data=>
-  {:id=>"4",
-   :type=>"job_application",
-   :attributes=>
-    {:position_title=>"Jr. CTO",
-      :date_applied=>"2024-10-31",
-      :status=>1,
-      :notes=>"Fingers crossed!",
-      :job_description=>"Looking for Turing grad/jr dev to be CTO",
-      :application_url=>"www.example.com",
-      :contact_information=>"boss@example.com",
-      :company_id=>35,
-      :company_name=>"Google"}}
+{
+  "data": {
+    "id": "3",
+    "type": "job_application",
+    "attributes": {
+      "position_title": "Backend Developer",
+      "date_applied": "2024-08-20",
+      "status": 2,
+      "notes": "Had a technical interview, awaiting decision.",
+      "job_description": "Developing RESTful APIs and optimizing server performance.",
+      "application_url": "https://creativesolutions.com/careers/backend-developer",
+      "company_id": 3,
```

Comment:
This could potentially influence my current ticket, I was planning on extracting the company name from the id. This could make the rendering easier in the long run, definitely behooves the app.